### DISCUSSION TITLE
Multiple lines on same path - overlapping each other

### DISCUSSION THREAD

**Discussion by Nitikaa** [visgl/deck.gl#7414] [Q&A]
Hi 

we have multiple layers on map of different types and those are overlapping each other and only the topmost one is visible on map.
Can you please suggest on how to can separate those so that all lines all visible and pickable on click.

**Comment by chrisgervang** [comment]
For paths, try `getOffset` to separate them. https://deck.gl/docs/api-reference/extensions/path-style-extension

**Reply by Nitikaa** [reply]
Can getOffset be used in all types of layers like linelayer and all ?

**Reply by sheeceg** [reply]
its for PolygonLayer and its derivatives like geojson layer maybe.

**Comment by sheeceg** [comment]
a code example to add off set to PolygonLayer based on a property:

```
        new PolygonLayer({
            ....         
            getOffset: d => d.putOffSet ? 5 : 0,
            extensions: [new PathStyleExtension({ dash: true, offset: true })]
        }),
```
